### ISSUE TITLE
How to change "CS_OPTION" id

### ISSUE THREAD

**Issue by kwayyinfotech** [Codestar/codestar-framework#163]
Hello,

I like to change CS_OPTION constant so my theme options will be saved using my theme's name. I set the constant in my functions.php but it's still saving with default  '_cs_options' name in db.

Here is my code:
```
define( 'CS_TEXTDOMAIN',  'mytheme' );
define( 'CS_OPTION',      'mytheme_options' );
if( !function_exists('cs_get_locale') ){
	require_once get_template_directory() . '/inc/codestar-framework/cs-framework.php';
}
```
I found this constant in cs-framework-path.php file.

Also I changed 'CS_TEXTDOMAIN' constant but I am not sure where it will be used. Please advice on this too.

Thanks

**Comment by Codestar** [comment]
Hi,

```CS_TEXTDOMAIN``` constant was removed. You are using older plugin files, if you see it. and ```CS_OPTION``` yes you can change it easy :wink: still saving with default. hmmm try to change it directly ```cs-framework-path.php``` still same ?

Codestar

**Comment by kwayyinfotech** [comment]
Hello,

Thanks for update. yes CS_TEXTDOMAIN is removed. I updated framework and checked it. 

For the CS_OPTION I checked it and found bug in my code. Actually the framework is used in plugin and theme both. I was adding the constant in theme's functions.php but the framework is init from plugin so theme's framework is not getting executed. Now I added constant in plugin's file and now it's working perfectly.

Thanks :)
